app/(api)/api/judges/post.js

```diff
@@ -0,0 +1,33 @@
+import { NextResponse } from 'next/server';
+import { ObjectId } from 'mongodb';
+
+import { getDatabase } from '@utils/db/mongoClient';
+import isBodyEmpty from '@utils/request/isBodyEmpty';
+import parseAndReplace from '@utils/request/parseAndReplace';
+
+export async function POST(request) {
+  try {
+    const body = await request.json();
+
+    if (isBodyEmpty(body)) {
+      throw new NoContentError('Empty request body.');
+    }
+    body.judge_pair_id = new ObjectId(body.judge_pair_id);
```

Comment:
Interesting choice to leave it up to the user to give a raw Id or use the convertID helper during the API call. This is nice